packages/compass-telemetry/src/generic-track.ts

```diff
@@ -0,0 +1,68 @@
+import { type Logger, mongoLogId } from '@mongodb-js/compass-logging';
```

Comment:
Totally get that it's hard to track this, but even importing mongoLogId here would actually mean bundling the whole logger package (this is why we re-implement this method in the logging/provder, I think there is a comment somewhere there saying something similar)

```suggestion
import { type Logger, mongoLogId } from '@mongodb-js/compass-logging/provider';
```